### ISSUE TITLE
Switch abandoned jakub-onderka/php-console-color to php-parallel-lint/php-console-color

### ISSUE THREAD

**Issue by Rohaq** [php-parallel-lint/PHP-Console-Highlighter#8]
Currently installing this pulls in Jakub's now abandoned version, rather than `php-parallel-lint/php-console-color`. This throws the following warning on running `composer install`:

```
Package jakub-onderka/php-console-color is abandoned, you should avoid using it. Use php-parallel-lint/php-console-color instead.
```

This should probably be switched over.

**Comment by duxthefux** [comment]
@Rohaq I think this has been done already in `master` but a release is still missing.

Could we get a release please? :)

**Comment by reedy** [comment]
That's https://github.com/php-parallel-lint/PHP-Console-Highlighter/issues/1

**Comment by jrfnl** [comment]
The [`0.5`](https://github.com/php-parallel-lint/PHP-Console-Highlighter/releases/tag/v0.5) release which was just tagged should fix this.

**Comment by reedy** [comment]
> The [`0.5`](https://github.com/php-parallel-lint/PHP-Console-Highlighter/releases/tag/v0.5) release which was just tagged should fix this, though it would still be better if the Console-Color package also saw a new tagged release.

https://github.com/php-parallel-lint/PHP-Console-Color/issues/1 and https://github.com/php-parallel-lint/PHP-Console-Color/pull/6

**Comment by grogy** [comment]
Fixed with [release 0.5](https://github.com/php-parallel-lint/PHP-Console-Highlighter/releases/tag/v0.5). Try it now, please
